argo/workflow.yaml

```diff
@@ -0,0 +1,142 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: clinvar-extraction-
+spec:
+  entrypoint: extract-latest-clinvar-release
+  podGC:
+    strategy: OnPodSuccess
+  volumeClaimTemplates:
+    - metadata:
+        name: state-dir
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            # ClinVar releases are typically ~650Mi when gzipped, and extract to
+            # ~7-8Gi. Throw in some extra for gsutil scratch space.
+            storage: 10Gi
```

Comment:
I ended up reverting back to a static volume size here. It's definitely possible to do the dynamic disk sizing by launching a sub-workflow, it's just more work than we need to do for ClinVar itself since their release payloads are always around the same size.

(Feel free to cut the same corner for the final Prefect workflow, once you're confident that the dynamic sizing is possible).